cliquet/resource.py

```diff
@@ -899,22 +899,35 @@ def _extract_filters(self, queryparams=None):
             value = native_value(value)
 
             # Ignore specific fields
-            if param.startswith('_') and param not in ('_since', '_to'):
+            if param.startswith('_') and param not in ('_since',
+                                                       '_to',
+                                                       '_before'):
                 continue
 
             # Handle the _since specific filter.
-            if param in ('_since', '_to'):
+            if param in ('_since', '_to', '_before'):
                 if not isinstance(value, six.integer_types):
                     error_details = {
                         'name': param,
                         'location': 'querystring',
-                        'description': 'Invalid value for _since'
+                        'description': 'Invalid value for %s' % param
                     }
                     raise_invalid(self.request, **error_details)
 
                 if param == '_since':
                     operator = COMPARISON.GT
                 else:
+                    if param == '_to':
+                        message = ('_to is now deprecated, '
+                                   'you should use _before instead')
+                        url = ('http://cliquet.rtfd.org/en/latest/api/resource'
+                               '.html#list-of-available-url-parameters')
+                        self.request.response.headers['Alert'] = encode_header(
+                            json.dumps({
+                                'code': 'soft-eol',
+                                'message': message,
+                                'url': url
+                            }))
```

Comment:
Ok I agree with you.

Yes, please add a helper (more like ``errors.send_alert(message)`` along with raise_invalid).

I propose that we don't put too much details in the url, and leave something generic like ``settings[documentation_url]``. Because for example, there is no guarantee that the version pointed on readthedocs is the one running on the server.